lib/std/c.zig

```diff
@@ -1527,6 +1527,18 @@ pub usingnamespace switch (native_os) {
     },
 };
 
+// clock_nanosleep() and TIMER_ABSTIME exist in the libc of only some Unix-like
+// systems, but they are consistent everywhere they exist.
+pub usingnamespace switch (native_os) {
```

Comment:
This is unlikely to get merged using `usingnamespace`, see #19214 for how to avoid using it.